<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>436</width>
    <height>278</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>436</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="enabledB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Włącz wtyczkę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="extsB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Ilośc bitów na próbkę</string>
   </property>
   <widget class="QRadioButton" name="u8B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>8bit unsigned char</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="s8B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>8bit signed char</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="i16B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>16bit integer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="i24B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>24bit integer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="i32B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>32bit integer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="f32B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>32bit float</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Próbkowanie</string>
   </property>
   <widget class="QSpinBox" name="srateB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4000</number>
    </property>
    <property name="maximum">
     <number>384000</number>
    </property>
    <property name="value">
     <number>4000</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Format odczytu</string>
   </property>
   <widget class="QRadioButton" name="beB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Big endian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="leB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Little endian</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Kanały</string>
   </property>
   <widget class="QSpinBox" name="chnB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Pomiń bajty</string>
   </property>
   <widget class="QSpinBox" name="skipB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>65536</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>extsB</tabstop>
  <tabstop>u8B</tabstop>
  <tabstop>s8B</tabstop>
  <tabstop>i16B</tabstop>
  <tabstop>i24B</tabstop>
  <tabstop>i32B</tabstop>
  <tabstop>f32B</tabstop>
  <tabstop>beB</tabstop>
  <tabstop>leB</tabstop>
  <tabstop>chnB</tabstop>
  <tabstop>srateB</tabstop>
  <tabstop>skipB</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
